notebooks/custom_kinfraglib/1_3_custom_filters_synthesizability.ipynb

```diff
@@ -0,0 +1,2349 @@
+{
```

Comment:
<p>Try to use more explicit terms for the links, e.g. instead of <code>method</code>  -&gt; (SYBA github repo)[https://github.com/lich-uct/syba]</p><p>Check grammar in last sentence.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/sonjaleo/KinFragLib/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>